workspace/notebook/py_vw_hist_green_specialist_case.json

```diff
@@ -0,0 +1,296 @@
+					"Prathap Adicherla &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;01-April-2025 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Casework Specialist curated view and table for Power BI"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"from pyspark.sql.functions import udf, col, lit, when, coalesce, concat, sum \n",
+					"from pyspark.sql.types import DateType, TimestampType, StringType, IntegerType, FloatType, DoubleType, BooleanType, StructType, StructField, ArrayType, MapType\n",
+					"from pyspark.sql.window import Window\n",
+					"from pyspark.sql.functions import row_number, rank, dense_rank, lag, lead, ntile\n",
+					"from datetime import datetime, timedelta\n",
+					"import pandas as pd\n",
+					"import numpy as np"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"%run utils/py_logging_decorator"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"%run utils/py_applicationinsights"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"spark_table_final = \"odw_curated_db.pbi_hist_green_specialist_case\"\n",
+					"view_name = \"odw_curated_db.vw_hist_green_specialist_case\"\n",
+					"source_table = \"odw_harmonised_db.hist_green_specialist_case\"\n",
+					"\n",
+					"start_exec_time = datetime.now()\n",
+					"insert_count = 0\n",
+					"update_count = 0\n",
+					"delete_count = 0\n",
+					"error_message=''\n",
+					"storage_account = ''"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"if not error_message:\n",
+					"    try:\n",
+					"        logInfo(\"Starting Casework Specialist curated table setup\")\n",
+					"        \n",
+					"        # Get storage account \n",
+					"        storage_account = mssparkutils.notebook.run('/utils/py_utils_get_storage_account')\n",
+					"        logInfo(f\"Using storage account: {storage_account}\")\n",
+					"        \n",
+					"        # Fix the path\n",
+					"        storage_account = storage_account.rstrip('/')\n",
+					"        delta_table_path = f\"abfss://odw-curated@{storage_account}/green_specialist_case/pbi_hist_green_specialist_case\"\n",
+					"        logInfo(f\"Delta table will be created at: {delta_table_path}\")\n",
+					"        \n",
+					"    except Exception as e:\n",
+					"        error_message = f\"Error in getting storage account configuration: {str(e)[:800]}\"\n",
+					"        end_exec_time = datetime.now()"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"if not error_message:\n",
+					"    try:\n",
+					"        # Create/refresh view\n",
+					"        logInfo(f\"Creating view {view_name}\")\n",
+					"        spark.sql(f\"\"\"\n",
+					"        CREATE OR REPLACE VIEW {view_name} AS\n",
+					"        SELECT  \n",
+					"            casework_specialist_id,\n",
```

Comment:
Just checking here your view it purely a pass through. if the goal is Power BI then maybe casting for dates/timestamps and having consistent naming if PBI expects it ?